### PR TITLE
Updated profile 

### PR DESCRIPTION
**Pull request by rghdrizzle**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4859"></a>

### CODE REVIEW

**Comment by SaraJaoude** [comment]
Please resolve conflicts.  I suggest you also update your fork as I suspect this is why conflicts have arisen.

**Review by Sindhuinti — DISMISSED** [review]

**Comment by rghdrizzle** [comment]
Resolved conflicts

File: data/rghdrizzle.json [hunk 1]
```diff
@@ -18,6 +18,11 @@
       "name": "Linkedin",
       "url": "https://www.linkedin.com/in/luqmaan-hakeem-57964a227/",
       "icon": "FaLinkedin"
+    },
+    {
+      "name": "Read my blogs",
+      "url": "https://rghdrizzle.hashnode.dev/",
+      "icon": "FaHashnode"
```
Comment (kumarsonsoff3) [inline data/rghdrizzle.json:25]: You're probably be using an invalid icon. It should be `SiHashnode`
```suggestion
      "icon": "SiHashnode"
```

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**